src/www/system_general.php

```diff
@@ -424,6 +426,20 @@
                 </div>
               </td>
             </tr>
+            <tr>
+              <td><a id="help_for_dhcpd6_opt" href="#" class="showhelp"><i class="fa fa-info-circle"></i></a> <?=gettext("Manual IPv6 DHCPD and RA"); ?></td>
+              <td>
+                <input name="dhcpd6track6allowoverride" type="checkbox" value="yes" <?= $pconfig['dhcpd6track6allowoverride'] ? 'checked="checked"' : '' ?>/>
```

Comment:
Shouldn't be global. Either we set this in the interface, or we don't set anything and figure it out from the dhcpv6 server side

Reply:
We could add it to the interface itself I suppose.